Sources/SwiftSyntaxBuilder/DeclBuildables.swift.gyb

```diff
@@ -185,3 +192,55 @@ public struct Struct: DeclBuildable {
     return DeclSyntax(structDecl)
   }
 }
+
+// MARK: Generated
+
+% for node in SYNTAX_NODES:
```

Comment:
Okay, this ended up pretty messy. :( 
Should we add some methods in `Node.py` or is this okay?
Maybe I've missed some methods that already is in the swift repository that could make it more clean?

Reply:
I was seeing we have a list here of [kinds here](https://github.com/apple/swift/blob/main/utils/gyb_syntax_support/kinds.py#L5).

We could loop over them and generate the code protocols

Reply:
Sounds good 👍